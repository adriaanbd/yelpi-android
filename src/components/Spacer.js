import React from 'react';
import {View} from 'react-native';

const Spacer = ({children, space}) => {
  const height = space || 25;
  return <View style={{height}}>{children}</View>;
};

export default Spacer;
