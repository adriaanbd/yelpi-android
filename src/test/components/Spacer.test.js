import { shallow } from 'enzyme';
import { View } from 'react-native';
import Spacer from '../../components/Spacer';

describe('Spacer', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<Spacer space={80} />);

    expect(wrapper.containsMatchingElement(<View />)).toBe(true)

    expect(wrapper).toMatchSnapshot();
  })
})