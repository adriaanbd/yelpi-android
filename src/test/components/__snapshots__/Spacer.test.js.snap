// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacer should render correctly 1`] = `
<View
  style={
    Object {
      "height": 80,
    }
  }
/>
`;
